### PR TITLE
BZ#2056737: Configure Ansible repository (Sat only)

### PR DESCRIPTION
**Pull request by Lennonka**
BZ: https://bugzilla.redhat.com/show_bug.cgi?id=2056737

This concerns a topic that is published for disconnected Satellite only.

Cherry-pick into:

* [x] Foreman 3.1

### CODE REVIEW

**Comment by ezr-ondrej** [comment]
I do not know enough about the process, but FWIW LGTM

File: guides/doc-Upgrading_and_Updating/topics/upgrading_a_disconnected_satellite.adoc [hunk 1]
```diff
@@ -119,6 +119,14 @@ Do not install or update any packages at this stage.
 ----
 baseurl=file:///media/sat6/
 ----
+... Optional: If you want to configure a custom path for the Ansible Runner packages, navigate to the `Ansible` repository and modify the `baseurl` directive with your custom path.
```
Comment (evgeni) [inline guides/doc-Upgrading_and_Updating/topics/upgrading_a_disconnected_satellite.adoc:122]: the BZ is about ansible, which needs to be set up identically to the SCL below (with the caveat, that there seem not to be an ansible.repo file on the ISO). and it's not optional.

Comment (Lennonka) [reply guides/doc-Upgrading_and_Updating/topics/upgrading_a_disconnected_satellite.adoc:122]: Thank you!

**Comment by Lennonka** [comment]
@evgeni Like this? :)

**Comment by evgeni** [comment]
almost, the repo file will now be missing some crucial entries (as there was no template to use from)

let me check if the missing file is intentional (then we need to properly document how to create one) or not (then we don't)

**Comment by Lennonka** [comment]
@evgeni Hi! Any updates? :)

**Comment by evgeni** [comment]
So far the update is "it shouldn't be like this", but also no clear path forward if that can be fixed trivially. :(

**Comment by Lennonka** [comment]
@evgeni We're going to provide a KBase article to give our users the workaround for the missing file and include a Known Issue in the RN for 6.10 -- this will be a separate PR. As for 7.0, this is going to be fixed and I believe we can go ahead with this and merge it.

**Comment by Lennonka** [comment]
@evgeni Oopsiedaisy, I missed that. Thanks, will do!

**Comment by Lennonka** [comment]
Naming confirmed with the ISO composition team.
@evgeni Will you do the honors? :)

File: guides/doc-Upgrading_and_Updating/topics/upgrading_a_disconnected_satellite.adoc [hunk 2]
```diff
@@ -106,20 +106,50 @@ Do not install or update any packages at this stage.
 # vi /etc/yum.repos.d/sat6.repo
 ----
 
-... Change the default `InstallMedia` repository name to `{Project}-{ProjectVersion}`:
+.. Change the default `InstallMedia` repository name to `{Project}-{ProjectVersion}`:
```
Comment (evgeni) [inline guides/doc-Upgrading_and_Updating/topics/upgrading_a_disconnected_satellite.adoc:109]: why this level change? these are (technically) sub-items of the "edit the repo file" section.

**Review by evgeni — COMMENTED** [review]
I am cool with the Ansible part, but unsure about the level change overall.

**Review by evgeni — APPROVED** [review]
I shall first read the comments… ACK

**Comment by Lennonka** [comment]
I've kept the list structure in the end.

**Comment by evgeni** [comment]
[3.2 496a28aa] BZ#2056737: Configure Ansible repository
[3.1 38b5d654] BZ#2056737: Configure Ansible repository

**Pull request merged**